cargo-shuttle/src/client.rs

```diff
@@ -182,12 +182,22 @@ impl Client {
         self.get(path).await
     }
 
-    pub async fn delete_project(&self, project: &ProjectName) -> Result<project::Response> {
+    pub async fn stop_project(&self, project: &ProjectName) -> Result<project::Response> {
         let path = format!("/projects/{}", project.as_str());
 
         self.delete(path).await
     }
 
+    pub async fn delete_project(&self, project: &ProjectName, dry_run: bool) -> Result<String> {
+        let path = format!(
+            "/projects/{}/delete{}",
+            project.as_str(),
+            if dry_run { "?dry_run=true" } else { "" }
```

Comment:
Good idea! Don't we need to add this to the clap args as well?

Reply:
c-s does a dry run before the actual call to check that business rules are fulfilled before the final deletion prompt.